src/ts/component/block/embed.tsx

```diff
@@ -645,6 +645,12 @@ const BlockEmbed = observer(class BlockEmbed extends React.Component<I.BlockComp
 						allowScript = !!text.match(/src="https:\/\/gist.github.com([^"]+)"/);
 					};
 
+					if (processor == I.EmbedProcessor.Sketchfab && text.match(/<(iframe|script)/)) {
+						const iframeReg = /<iframe.*?<\/iframe>/;
+						const iframeStr = text.match(iframeReg)?.[0] || '';
+						text = iframeStr;
+					};
```

Comment:
- Logic conditions should be wrapped in parenthesis `(processor == I.EmbedProcessor.Sketchfab)`
- Make line break after const iframeStr
- I do not perfectly understand how it should work

Reply:
The original embedded text may contain some remaining HTML structure, which is to extract the iframe inside. After all, not all users will delete the extra parts.
```html
<div class="sketchfab-embed-wrapper"> <iframe title="Steamboat Willie" frameborder="0" allowfullscreen mozallowfullscreen="true" webkitallowfullscreen="true" allow="autoplay; fullscreen; xr-spatial-tracking" xr-spatial-tracking execution-while-out-of-viewport execution-while-not-rendered web-share src="https://sketchfab.com/models/58c5716c49c947f2a5aa152cec39cec0/embed"> </iframe> <p style="font-size: 13px; font-weight: normal; margin: 5px; color: #4A4A4A;"> <a href="https://sketchfab.com/3d-models/steamboat-willie-58c5716c49c947f2a5aa152cec39cec0?utm_medium=embed&utm_campaign=share-popup&utm_content=58c5716c49c947f2a5aa152cec39cec0" target="_blank" rel="nofollow" style="font-weight: bold; color: #1CAAD9;"> Steamboat Willie </a> by <a href="https://sketchfab.com/tramdrey?utm_medium=embed&utm_campaign=share-popup&utm_content=58c5716c49c947f2a5aa152cec39cec0" target="_blank" rel="nofollow" style="font-weight: bold; color: #1CAAD9;"> tramdrey </a> on <a href="https://sketchfab.com?utm_medium=embed&utm_campaign=share-popup&utm_content=58c5716c49c947f2a5aa152cec39cec0" target="_blank" rel="nofollow" style="font-weight: bold; color: #1CAAD9;">Sketchfab</a></p></div>
```

Reply:
> * Logic conditions should be wrapped in parenthesis `(processor == I.EmbedProcessor.Sketchfab)`
> * Make line break after const iframeStr
> * I do not perfectly understand how it should work

However, I thought again that it seems better to give the initiative to the user.